cmd/cloudflared/tunnel/configuration.go

```diff
@@ -348,6 +348,18 @@ func prepareTunnelConfig(
 	if err != nil {
 		return nil, nil, err
 	}
+	edgeBindAddr, err := parseConfigBindAddress(c.String("edge-bind-address"))
+	if err != nil {
+		return nil, nil, err
+	}
+	if err := testIPBindable(edgeBindAddr); err != nil {
```

Comment:
This `edgeBindAddr` can be `nil` as returned by `parseConfigBindAddress`. Should probably be checked before attempting to test the provided IP address.

Reply:
Updated `testIPBindable` to simply accept `nil`.